portal-web/test/functional/com/liferay/portalweb/tests/enduser/analyticscloud/PageEvents.testcase

```diff
@@ -819,16 +819,52 @@ definition {
 			var expectedDataSourceId = ACSettings.getDXPInstanceId(dataSourceName = "${dataSourceName}");
 		}
 
+		task ("Add a new site with a public widget page") {
+			ACUtils.addPage(layoutName = "AC Page 2");
+		}
+
+		task ("Navigate to the site page") {
+			ACUtils.navigateToSitePage(
+				pageName = "AC Page 2",
+				siteName = "Site Name");
+		}
+
+		task ("Start Har recording") {
+			ProxyUtil.startHarRecording("tabBlurred");
+		}
+
 		task ("Navigate to the site page") {
 			ACUtils.navigateToSitePage(
 				pageName = "AC Page",
 				siteName = "Site Name");
 		}
 
+		task ("Wait for sending requests") {
+			ACUtils.waitForSendingRequest();
+		}
+
+		task ("View the tabBlurred event not triggered") {
+			ACUtils.assertEventNotPresent(
+				applicationId = "Page",
+				eventId = "tabBlurred");
+		}
+
 		task ("Open the BBC home page in a new tab") {
 			Open.openInTheNewTab(locator1 = "https://www.bbc.com");
 		}
 
+		task ("Switch to the previous tab") {
+			SelectWindow(locator1 = "title=AC Page - Site Name - ${dataSource}");
+		}
+
+		task ("Hard refresh the page to clear the network") {
+			TestUtils.hardRefresh();
+		}
+
+		task ("Switch to the BCC tab") {
+			SelectWindow(locator1 = "title=BBC - Homepage");
+		}
+
 		task ("Start Har recording") {
```

Comment:
Hey @ViniLopes20 thanks for working on this! Do we need this harRecording step anymore? Since you already added another harRecording before navigating to AC Page. Or are you keeping this for the second scenario where you switch to an untracked site?

Reply:
Hey @calvinctvo, I put _harRecording_ again, because the test was not finding the event that was being launched, it gave an error in Java. So when I put the _harRecording_ again before the event was launched, the test worked. This second _harRecording_ is to start recording the _tabBlurred_ event that will appear when you return to the Liferay DXP page.

Reply:
Hey @ViniLopes20 , so you would need _harRecording_ for every event that you need to check?

Reply:
I don't think for every event, but I think with `hardRefresh`, I think it somehow disturb the working of `harRecording` 🤔

Reply:
Thanks @ViniLopes20 for the explanation. Everything else LGTM